### DISCUSSION TITLE
Breaking change in 3.48.0

### DISCUSSION THREAD

**Discussion by eraffel-MDSol** [jeremyevans/roda#240] [General]
This seems to be a breaking change. It changes the return value of `freeze` from `super` to the value of the `instance_variable_get` call:
https://github.com/jeremyevans/roda/commit/e96f39f21bd67ffd488b51ebf89416e6b2d84a90#diff-175993f96ccd9dd88245f1f516372183dfa6328b293d55b631c1267de2afbdcaL147

**Comment by jeremyevans** [comment]
Yep, that's a bug.  I'll fix it right away.

**Reply by jeremyevans** [reply]
Fixed by ac28385e45e48d76a043df6e472dbf9590734391

**Reply by eraffel-MDSol** [reply]
Thanks! Any idea when you might push a tag. I can just watch the repo if not, but if it might be a while I just want to update our gemfile to make sure we don't accidentally upgrade to a bugged version.

**Comment by jeremyevans** [comment]
Next release (3.52.0) will be tomorrow and will include this fix.
